### PR TITLE
Fixes for Create (Fabric) / Flywheel

### PR DESCRIPTION
**Pull request by IMS212**
Current fixes:
Change level renderer buffers during shadow rendering
~~Iris API: getShadowFrustum (null when not rendering shadows)~~ removed for now

### CODE REVIEW

File: src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java [hunk 1]
```diff
@@ -518,6 +518,9 @@ public void renderShadows(LevelRendererAccessor levelRenderer, Camera playerCame
 		profiler.popPush("shadows");
 		ACTIVE = true;
 
+		RenderBuffers playerBuffers = levelRenderer.getRenderBuffers();
```
Comment (coderbot16) [inline src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java:521]: Could we add a comment that this is needed for mods like Flywheel? Just good to note down why this code is needed so we don't delete it in a refactor.

Comment (IMS212) [reply src/main/java/net/coderbot/iris/pipeline/ShadowRenderer.java:522]: done

**Comment by coderbot16** [comment]
This seems to conflict with the shadow rendering changes that I just merged.

**Comment by IMS212** [comment]
Fixed

**Comment by IMS212** [comment]
Do you think we can get this merged before 1.2.1?

**Comment by coderbot16** [comment]
Yes I do believe we can get it merged before then :P

**Pull request merged**